Library/Homebrew/cmd/test-bot.rb

```diff
@@ -91,6 +92,13 @@ def failed?
 
     def puts_command
       cmd = @command.join(" ")
+      if ENV["TRAVIS"]
+        puts cmd
+        @travis_timer_id = rand(2**32).to_s(16)
```

Comment:
Should we use the checksum of commands instead of random value which may reproduce the same value.